siliconcompiler/tools/yosys/syn_fpga.tcl

```diff
@@ -4,19 +4,157 @@ set job_name [dict get $sc_cfg option jobname]
 set step [dict get $sc_cfg arg step]
 set index [dict get $sc_cfg arg index]
 
-#TODO: add logic that remaps yosys built in name based on part number
+set sc_memmap_techlib "None"
+set sc_mem_techlib "None"
+set sc_lut_techlib "None"
+set sc_dsp_techlib "None"
+set sc_flop_techlib "None"
```

Comment:
what are these used for?

Reply:
These are placeholders that are temporarily disabling the flow's ability to perform technology mapping to user-defined tech libraries for each of the major hard block types in modern FPGAs.  Based on other feedback I've received on this PR, I'm guessing you're going to want me to strip down the syn_fpga.tcl and my testcases to eliminate all of this until I have a test case that performs technology mapping?

Reply:
Per above comment, I've implemented the reduced syn_fpga.tcl.  We can add things back at such as a time as I am able to furnish an architecture with hard macros that require technology mapping.